cmd/context/delete.go

```diff
@@ -28,40 +30,47 @@ import (
 func DeleteCMD() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete",
-		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
-		Short: "Delete a context",
+		Args:  utils.MinimumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
+		Short: "Delete one or more contexts",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			args[0] = okteto.AddSchema(args[0])
-			args[0] = strings.TrimSuffix(args[0], "/")
-			return Delete(args[0])
+			for idx, arg := range args {
+				args[idx] = okteto.AddSchema(arg)
+				args[idx] = strings.TrimSuffix(arg, "/")
+			}
+			analytics.TrackContextDelete(len(args))
+			return Delete(args)
```

Comment:
If I run `okteto ctx delete foo foo foo foo foo foo foo foo foo foo foo` the track event will report `success: true` and `totalContextsDeleted: 11` which is misleading.

To maintain integrity, we could:
- `errs := Delete(args)`
- track event after deletion
- pass to `totalContextsDeleted = len(args) - len(errs)`
- if `len(args) == len(errs)` then pass `success: false`
- `return errs`

Reply:
@jeremyphua will you look into the above? if you don't have time, just let me know and I can follow up, cheers!

Reply:
Hi @AdrianPedriza, I've made the changes in the latest commit! But I'm not too sure if it is acceptable.